src/gather/tests/test_commands.py

```diff
@@ -31,6 +31,23 @@ def main4(args):
 def weird_function():
     pass
 
+CONFLICTING_COMMANDS = gather.Collector()
```

Comment:
I understand `venusian` makes this relatively safe, but would it be possible to put these into the `setUp` method?  If mutation isn't a concern, sharing fixtures between tests might be.